packages/application-shell/src/utils/location/location.ts

```diff
@@ -0,0 +1,7 @@
+const replace = (url: string): void => window.location.replace(url);
```

Comment:
The idea is we wrap more APIs we use from the window's location when we need it.